### PR TITLE
BROKERS: Navigations Added

### PR DESCRIPTION
**Pull request by NaynaMallick**

### CODE REVIEW

File: ArtGallery.Web.Api/Startup.cs [hunk 1]
```diff
@@ -15,10 +16,9 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddRazorPages();
             services.AddServerSideBlazor();
-            services.AddTransient<IDateTimeBroker, DateTimeBroker>();
-
-            services.AddRazorPages(options =>
-                options.RootDirectory = "/Views/Pages");
+            AddRootDirectory(services);
+            services.AddScoped<IDateTimeBroker, DateTimeBroker>();
```
Comment (ShriHumrudha) [inline ArtGallery.Web.Api/Startup.cs]: I gave this comment in the previous PR to group the brokers.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ArtGallery.Web.Api/Startup.cs (hunk 1 above)
Comment (ShriHumrudha) [reply ArtGallery.Web.Api/Startup.cs]: Any specific reason on this closure?

File: ArtGallery.Web.Api/Brokers/Navigations/NavigationBroker.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+namespace ArtGallery.Web.Api.Brokers.Navigations
+{
+    public class NavigationBroker : INavigationBroker
+    {
+        private readonly NavigationManager navigationManager;
+
+        public NavigationBroker(NavigationManager navigationManager) =>
+            this.navigationManager = navigationManager;
+
+        public void NavigateTo(String route) =>
```
Comment (ShriHumrudha) [inline ArtGallery.Web.Api/Brokers/Navigations/NavigationBroker.cs]: Use string data type than the String class

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ArtGallery.Web.Api/Brokers/Navigations/NavigationBroker.cs (hunk 2 above)
Comment (NaynaMallick) [reply ArtGallery.Web.Api/Brokers/Navigations/NavigationBroker.cs]: resolved

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**